testapp/src/main/java/com/rakuten/tech/mobile/testapp/ui/display/MiniAppDisplayActivity.kt

```diff
@@ -151,10 +150,16 @@ class MiniAppDisplayActivity : BaseActivity() {
     private fun setupMiniAppMessageBridge() {
         // setup MiniAppMessageBridge
         miniAppMessageBridge = object : MiniAppMessageBridge() {
+
             override fun getUniqueId(
                     onSuccess: (uniqueId: String) -> Unit,
                     onError: (message: String) -> Unit
-            ) = onSuccess(AppSettings.instance.uniqueId)
+            ) {
+                onSuccess(AppSettings.instance.uniqueId)
```

Comment:
Right now this is always sending success, so it needs to be moved below to the `if/else` which is checking if there's an error.

Reply:
Yeah, I have updated this part.